front/public/locales/fr/errors.json

```
@@ -214,8 +214,8 @@
       "TowedRollingStockNotFound": "Matériel roulant remorqué {towed_rolling_stock_id} non trouvé",
       "TrainSimulationFail": "Échec de la simulation du train",
       "TimetableNotFound": "Grille horaire '{{timetable_id}}' non trouvée",
-      "InvalidConsistMass": "La masse du convoi est invalide : '{{message}}'",
-      "InvalidConsistLength": "La longueur du convoi est invalide : '{{message}}'"
+      "InvalidConsistMass": "La masse du convoi doit être supérieure à '{{min}}'",
+      "InvalidConsistLength": "La longueur du convoi doit être supérieure à '{{min}}'"
```

To match the 🇬🇧 part (we may do the opposite and change 🇬🇧 locale + phrase of the code):
```suggestion
      "InvalidConsistMass": "Masse du convoi invalide : elle doit être supérieure à '{{min}}'",
      "InvalidConsistLength": "Longueur du convoi invalide : doit être supérieure à '{{min}}'"
```